src/docs/product/accounts/pricing.mdx

```diff
@@ -46,8 +46,9 @@ You can set a maximum monthly on-demand invoice amount by setting an on-demand b
 
 <Note>
 
-Per-category on-demand budgets are only available on plans with [performance monitoring](/product/performance/) features and if you're in the Early Adopter program.
-If you opt out of the Early Adopter program while on the per-category on-demand budget strategy, you'll retain the per-category budgets you've set and the ability to update them. However, if you switch to the shared on-demand budget strategy, you won't be able to switch back unless you opt in to the Early Adopter program again.
+Per-category on-demand budgets are rolling out for general availability to paying customers on plans with [performance monitoring](/product/performance/) features.
+
+In addition, it is available if you're opted into the Early Adopter program. If you opt out of the Early Adopter program while on the per-category on-demand budget strategy, you'll retain the per-category budgets you've set and the ability to update them. However, if you switch to the shared on-demand budget strategy, you won't be able to switch back unless you opt in to the Early Adopter program again.
```

Comment:
Does this mean EA customers on any plan? If so, suggest:
```suggestion
In addition, it is available if you're opted into the Early Adopter program on any plan. If you opt out of the Early Adopter program while on the per-category on-demand budget strategy, you'll retain the per-category budgets you've set and the ability to update them. However, if you switch to the shared on-demand budget strategy, you won't be able to switch back unless you opt in to the Early Adopter program again.
```

Reply:
@imatwawana We don't support on-demand budgets on legacy plans. Your suggestion might imply that.

Reply:
New suggestion (I've also made a suggestion for the preceding sentence to support this edit):
```suggestion
Until the GA rollout is complete, the feature is still available for organizations on paid performance plans in the Early Adopter program. If you opt out of the Early Adopter program while on the per-category on-demand budget strategy, you'll retain the per-category budgets you've set and the ability to update them. However, if you switch to the shared on-demand budget strategy, you won't be able to switch back unless you opt in to the Early Adopter program again.
```